rclcpp/include/rclcpp/parameter.hpp

```diff
@@ -224,7 +224,7 @@ class ParameterVariant
     return parameter;
   }
 
-  std::string to_string() const
+  std::string value_to_string() const
```

Comment:
This method seems to only operate on the `ParameterValue`. Why is it provided in the `ParameterVariant` class?